routes/web.php

```diff
@@ -2,14 +2,18 @@
 
 use App\Http\Controllers\PostController;
 use Illuminate\Support\Facades\Route;
-use App\Http\Controllers\PageController;
+use App\Http\Controllers\StaticPageController;
 
-Route::get('/', [PageController::class, 'welcome'])->name('welcome');
-Route::get('/about', [PageController::class, 'profile'])->name('profile');
-Route::get('/dashboard', [PageController::class, 'dashboard'])->name('dashboard');
-Route::get('/faq', [PageController::class, 'faq'])->name('faq');
+Route::get('/', [StaticPageController::class, 'welcome'])->name('welcome');
+Route::get('/about', [StaticPageController::class, 'profile'])->name('profile');
+Route::get('/dashboard', [StaticPageController::class, 'dashboard'])->name('dashboard');
+Route::get('/faq', [StaticPageController::class, 'faq'])->name('faq');
 
 Route::get('/posts', [PostController::class, 'index'])->name('posts');
 Route::get('posts/create', [PostController::class, 'create'])->name('posts.create');
 Route::post('posts', [PostController::class, 'store'])->name('posts.store');
 Route::get('posts/{post}', [PostController::class, 'show'])->name('posts.show');
+Route::get('posts/{post}/edit', [PostController::class, 'edit'])->name('posts.edit');
+Route::put('posts/{post}', [PostController::class, 'update'])->name('posts.update');
+Route::get('posts/{post}/delete', [PostController::class, 'delete'])->name('posts.delete');
+Route::delete('posts/{post}', [PostController::class, 'destroy'])->name('posts.destroy');
```

Comment:
You can register all 7 RESTful routes in just one command. It will save you a lot of time.